src/util.cc

```diff
@@ -432,9 +432,8 @@ std::wstring Utf8ToWide(const StringPiece& u8_string) {
   // Make sure that the string size can be casted to an int.
   assert(!(u8_string.size() > INT_MAX));
```

Comment:
Need to use std::numeric_limits. "INT_MAX" is a C-ism that doesn't capture things properly.

assert(u8_string.size() <= std::numeric_limits<int>::max());